src/pages/steps/AdditionalInformationStep/DescriptionStep.tsx

```diff
@@ -159,6 +165,33 @@ const DescriptionStep = ({
     });
   };
 
+  const handleNewlinesAsSoftline = (event, editorState) => {
+    const selection = editorState.getSelection();
+
+    if (selection.isCollapsed()) {
+      setEditorState(RichUtils.insertSoftNewline(editorState));
+    } else {
+      const content = editorState.getCurrentContent();
+      let newContent = Modifier.removeRange(content, selection, 'forward');
+      const newSelection = newContent.getSelectionAfter();
+      const block = newContent.getBlockForKey(newSelection.getStartKey());
+
+      newContent = Modifier.insertText(
+        newContent,
+        newSelection,
+        '\n',
+        block.getInlineStyleAt(newSelection.getStartOffset()),
+        null,
+      );
+
+      setEditorState(
+        EditorState.push(editorState, newContent, 'insert-fragment'),
+      );
+    }
+
+    return true;
```

Comment:
Is it necessary to add the  `return true` here? 
```suggestion
```

Reply:
It has special meaning in Draft to indicate whether the newline was "handled" or whether the native behavior needs to kick in in addition to the callback. It's basically an `event.preventDefault()`

Reply:
Ok clear